media/js/common/buckets.js

```diff
@@ -25,7 +25,7 @@ define('buckets', [], function() {
         // Camel-case it.
         property = property[0].toUpperCase() + property.substr(1);
 
-        for (var i = 0, e; e = prefixes[i++];) {
+        for (var i = 0; e; e = prefixes[i++]) {
```

Comment:
This doesn't look right to me. Running this gives a ReferenceError e is not defined.

I think we should remove the assignment. Here's why this sort of thing is bad. http://jsbin.com/macuya/1/watch?js,console